app/controllers/dashboard_controller.rb

```diff
@@ -12,11 +12,11 @@ def edit
   end
 
   def update
-    @user = User.find(current_user.id)
-    city = Geocoder.search(params["city"]).first
-    current_user.lat = city.coordinates[0]
-    current_user.lng = city.coordinates[-1]
-    current_user.save!
-    redirect_to "/dashboard"
+    # @user = User.find(current_user.id)
+    # city = Geocoder.search(params["city"]).first
```

Comment:
If this isn't needed anymore can we get rid of it rather than just commenting it out?